packages/schema-type/src/typescript/index.ts

```diff
@@ -0,0 +1,10 @@
+export type { StaticSchemaType } from "./StaticSchemaType.js";
+export type { DynamicSchemaType } from "./DynamicSchemaType.js";
+export type { ArraySchemaType } from "./ArraySchemaType.js";
+export type { SchemaTypeToPrimitive } from "./SchemaTypeToPrimitive";
```

Comment:
looks like this makes the CI fail:
```suggestion
export type { SchemaTypeToPrimitive } from "./SchemaTypeToPrimitive.js";
```

Reply:
yep, just fixed!